packages/testcontainers/src/types.ts

```diff
@@ -81,6 +81,8 @@ export type RegistryConfig = {
 
 export type BuildArgs = { [key in string]: string };
 
+export type ExecOptions = { workingDir: string; user: string; env: Environment };
```

Comment:
I wanted to respect the original public API as much as possible to avoid any undesired issues. So, following what already existed, I opted not to add a log option here in the "public" API.